### ISSUE TITLE
[20220821] Weekly AI ArXiv 만담 - 63회차

### ISSUE THREAD

**Issue by jungwoo-ha** [jungwoo-ha/WeeklyArxivTalk#63]
### News
- Conferences
  - COLING 2022 decision notification: 경주에서 만나요!! (https://coling2022.org/index)
  - AAAI 2022 제출 모두 수고 하셨습니다.
  - [NeurIPS 2022 Workshop List](https://neurips.cc/virtual/2022/events/workshop)
    - [Human-centered AI WS](https://hcai-at-neurips.github.io/site/call.html)
- [인공지능대학원 심포지엄](https://youtu.be/8ZCPxEUvtCI)
- [슈퍼컴퓨터 6호기 도입한다..구축사업 예타 통과](https://n.news.naver.com/mnews/article/018/0005296640?sid=105)
- [2022 AI 대학원 심포지엄' 개최…국내 최고 AI 석학들 한 자리에](https://n.news.naver.com/mnews/article/003/0011369788?sid=105)
- [3행시 by TUNiB LM](https://tunibridge.ai/nverse)
- 모두연 7주년 홈커밍데이: 다음주에 김영도님이 ㅎㅎ

### ArXiv
- [Understanding Scaling Laws for Recommendation Models](https://arxiv.org/abs/2208.08489)
  - Meta에서 나온 추천시스템에서의 Scaling law 분석
  - MLP + Embedding 기반의 DLRM 이 주요 타겟 (https://arxiv.org/abs/2104.05158, ISCA 2022)
  - 특이한 것은 추천시스템의 Scaling시에 deploy와 운영까지 고려한 ROI를 위해 efficiency 관점에서 분석 (논문에서 ROI란 용어를 보게 될 줄이야..)
  - 주요 팩터는 역시 parameter, data, compute, data와 compute는 유사한 패턴인데 parameter 스케일업할때는 효율성 양상이 좀 다름
  - MLP 기반의 architecture 를 이용하는 추천시스템은 참조해 볼만
![image](https://user-images.githubusercontent.com/11782739/185789235-b35c5ae5-d527-40fc-ad8d-2d2408eb8b41.png)
![image](https://user-images.githubusercontent.com/11782739/185775890-869ff0c6-f8ed-45a9-acf9-408feef241c9.png)
![image](https://user-images.githubusercontent.com/11782739/185775902-8ea51917-90ea-4748-8b87-98ebd149e005.png)
![image](https://user-images.githubusercontent.com/11782739/185775912-717efea8-efde-496a-a02f-1f7c1dc7a60d.png)

- 흥미있는 연구들
  - [Musika! Fast Infinite Waveform Music Generation](https://arxiv.org/abs/2208.08706)
    - 직접 들어보시는 걸로! https://marcoppasini.github.io/musika (ISMIR 2022)
  - [Paint2Pix: Interactive Painting based Progressive Image Synthesis and Editing](https://arxiv.org/abs/2208.08092)
    - 사용자 컨트롤 가능한 이미지 합성 & 편집 (ECCV 2022, https://github.com/1jsingh/paint2pix)
  - [Understanding Long Documents with Different Position-Aware Attentions](https://arxiv.org/abs/2208.08201)

**Comment by laray5959** [comment]
### Arxiv (Audio and Speech Processing)

- [Speech Representation Disentanglement with Adversarial Mutual Information Learning for One-shot Voice Conversion](https://arxiv.org/pdf/2208.08757.pdf)
- [Disentangled Speaker Representation Learning via Mutual Information Minimization](https://arxiv.org/pdf/2208.08012.pdf)
- 흥미있는 연구들 
    - [Uconv-Conformer: High Reduction of Input Sequence Length for End-to-End Speech Recognition](https://arxiv.org/pdf/2208.07657.pdf)
    - [Differentiable WORLD Synthesizer-based Neural Vocoder With Application To End-To-End Audio Style Transfer](https://arxiv.org/pdf/2208.07282.pdf)
    - [DDX7: Differentiable FM Synthesis of Musical Instrument Sounds](https://arxiv.org/pdf/2208.06169.pdf)

**Comment by hollobit** [comment]
### "기계가 왜 떨어트렸지" AI면접 잣대 몰라 취준생들 한숨
https://news.v.daum.net/v/20220820000129514
- AI역량검사를 도입한 기업은 총 630곳. 2019년(260곳) 대비 2배 이상 증가했고, 기업들의 재계약률은 지난해 기준 93%
- AI 전형은 크게 AI역량검사와 AI면접 두 가지
- AI역량검사는 대개 게임 형식으로 이뤄지는데 게임에 대한 반응을 수집하고 무의식적 행동 패턴을 분석
- AI면접은 주로 소통역량을 평가. 대면 업무가 많은 영업·경영지원·서비스 업종에서 주로 활용. 얼굴 표정에서 감정을 추론하고, 목소리 높낮이나 말의 휴지(休止), 사용 언어 등을 분석
- 잡코리아에서 진행한 설문조사에 따르면 남녀 취업준비생 873명 중 71.9%가 ‘AI면접을 준비하고 있지 않다’고 했고, 그 이유로 ‘AI면접을 어떻게 준비해야 하는지 모르겠어서’라는 응답이 과반수(52.1%)를 차지
- ‘AI면접의 평가 방식과 진행 방법 등에 대해 알고 있는지’ 묻는 질의에 61.5%가 ‘잘 모른다’고 답. 
- 그러나 AI면접 도입 기업 52개 중 59.6%(31곳)가 면접 결과를 채용에 반영했고, 이 중 24%(6곳)는 ‘당락을 좌우한다’고 답변 
<img src="https://img2.daumcdn.net/thumb/R658x0.q70/?fname=https://t1.daumcdn.net/news/202208/20/joongangsunday/20220820000134991sixe.jpg">

### The AI World Government Conference (10/6 ~ 10/7) 
https://www.aiworldgov.com/
https://www.aitimes.kr/news/articleView.html?idxno=25805 
- 36개 미국 연방 정부 기관의 AI 정책 책임자들이 모이는 AI World Government가 10월6일과 7일 워싱턴 DC에서 개최 
- AI책임 모델 및 프레임워크, AI확장성, 인간 중심 AI, 투자 비즈니스 사례, AI 리더십 및 혁신, AI 및 지속 가능성, 인재 및 변화 관리, 사이버 보안 및 위험 관리, AI윤리 및 표준, 데이터 플랫폼 및 관리, 클라우드 및 AI, 아키텍처 및 인프라, ML/AI 및 IoT, 상호 운용성, AI 소프트웨어 공학, 대화형 AI, 일의 미래 등의 주제로 진행

**Comment by nick-jhlee** [comment]
## Papers
- ~~양심없는~~ 저의 preprint 홍보: [Nearly Optimal Latent State Decoding in Block MDPs](https://arxiv.org/pdf/2208.08480.pdf)
   - KAIST AI (Junghyun Lee*, Se-Young Yun), KTH (Yassir Jedra*, Alexandre Proutiere)
   - Block MDP를 *function approx*를 사용하지않고 이론적으로 분석한 첫 논문
   - model estimation i.e. clustering: lower bound + (near-matching) upper bound on error rate
   - reward-free RL: lower bound + (near-matching) upper bound on sample complexity

- Transformer is all you need, literally....: [Towards Learning Universal Hyperparameter Optimizers with Transformers](https://arxiv.org/pdf/2205.13320.pdf)
   - DeepMind, Google Research, Brain Team
   - Blogpost: https://ai.googleblog.com/2022/08/optformer-towards-universal.html
   - 배경: [Google Vizier: A Service for Black-Box Optimization](https://dl.acm.org/doi/pdf/10.1145/3097983.3098043)
   - **OptFormer**: first [Transformer](https://ai.googleblog.com/2017/08/transformer-novel-neural-network.html)-based frameworks for hyperparameter tuning, learned from large-scale optimization data using flexible text-based representations.
   - 옛날 history를 text로 받아서 하는 신박(?)한 방법
      - ex. “CIFAR”, “learning rate”, “optimizer type”, and “Accuracy”
 

- [Mining Legal Arguments in Court Decisions](https://arxiv.org/pdf/2208.06178.pdf)
   - Technical University of Darmstadt, Goethe-Universit¨at Frankfurt am Main
   - **LAM:ECHR**: "the uniquest and largest expert annotated corpus of legal arguments"
